### PR TITLE
Remove old experiments

### PR DESCRIPTION
**Pull request by iamtrask**
Our official policy is to remove things from the experimental repo
after a month of inactivity.

### CODE REVIEW

**Comment by gmuraru** [comment]
It would be a good idea to move this files to a separate branch called ```experimental```? And there we can try to run experiments before merging them into ```dev```.
Thinking about this again: we could have the experiments on our personal fork version and make a PR into ```dev``` when we know that they work.

File: examples/experimental/README.md [hunk 1]
```diff
@@ -1 +0,0 @@
-All notebooks put in this folder will be deleted if left unattended for 14 days or more. They exist purely for experimental purposes (and the ability to temporarily communicate experimental results to the group). As such, they are NOT guaranteed to work or checked for code style/functionality in any way. For more robust/tested/permanent notebooks, see other folders outside this one.
```
Comment (lc0) [inline examples/experimental/README.md]: So not just clean up of old experiments, but also removing overall concept of experiments. Is it correct?

Comment (iamtrask) [reply examples/experimental/README.md]: The directory will come back when someone puts in an experiment

**Comment by iamtrask** [comment]
> It would be a good idea to move this files to a separate branch called `experimental`? And there we can try to run experiments before merging them into `dev`.
> Thinking about this again: we could have the experiments on our personal fork version and make a PR into `dev` when we know that they work.

Agreed

File: examples/experimental/README.md (hunk 1 above)
Comment (lc0) [reply examples/experimental/README.md]: shall we keep readme in this case?

File: examples/experimental/Large Precision Tensor.ipynb [hunk 2]
```diff
@@ -1,411 +0,0 @@
-{
```
Comment (LaRiffle) [inline examples/experimental/Large Precision Tensor.ipynb]: Keep this one please it's still needed

File: examples/experimental/PATE Analysis.ipynb [hunk 3]
```diff
@@ -1,126 +0,0 @@
-{
```
Comment (LaRiffle) [inline examples/experimental/PATE Analysis.ipynb]: I'm not sure we want to remove our tutorial on PATE

File: examples/experimental/README.md (hunk 1 above)
Comment (LaRiffle) [reply examples/experimental/README.md]: I would agree to do so

File: examples/experimental/plaintext_speed_regression.ipynb [hunk 4]
```diff
@@ -1,1232 +0,0 @@
-{
- "cells": [
```
Comment (LaRiffle) [inline examples/experimental/plaintext_speed_regression.ipynb]: I'm sure André needs it while he's working on his grant

File: examples/experimental/PATE Analysis.ipynb (hunk 3 above)
Comment (LaRiffle) [reply examples/experimental/PATE Analysis.ipynb]: My bad, it's empty :(

**Comment by iamtrask** [comment]
Closing - the notebooks are getting more use than i expected

**Pull request closed without merging**